resources/sass/app.scss

```diff
@@ -24,8 +24,31 @@
 	width: 45%;
 }
 
+#ext-image-2{
+	height: 20%;
+	width: 100%;
+	background-color: green;
+	min-width: 320px;
+}
 
-//LIST PAGE ELEMENT HEIGHTS
+#ext-image-3{
+	height: 300px;
+	width: 300px;
+	background-color: yellow;
+}
+
+#ext-button-11{
+	top: 40%;
+	width: 50px;
+	height: 10%;
+}
```

Comment:
Targeting specific dom-element ids on a generated image or list of images worries me. Can we prove that the images we display are ext-image-1, ext-image-2, ext-image-3 every time?